skrifa/src/outline/glyf/deltas.rs

```diff
@@ -11,11 +11,21 @@ use read_fonts::{
 
 use super::PHANTOM_POINT_COUNT;
 
+#[derive(Copy, Clone, Debug)]
+pub(super) enum AvailableVarMetrics {
+    /// Full variable metrics with advances and side-bearings.
+    Present,
+    /// Variable advances but not side-bearings.
+    MissingSideBearings,
+    /// No variable metrics table.
+    Missing,
+}
```

Comment:
I think I would expect these to be named for what *is* available not what isn't? - e.g. All, Advances, None as opposed to Present/MissingSide/Missing